libraries/classes/Controllers/CheckRelationsController.php

```diff
@@ -43,7 +44,7 @@ public function __invoke(ServerRequest $request): void
 
         // If request for creating all PMA tables.
         if (isset($fixAllPmaDb)) {
-            $this->relation->fixPmaTables($GLOBALS['db']);
+            $this->relation->fixPmaTables(DatabaseName::fromValue($GLOBALS['db']));
```

Comment:
This method is expecting a string, not a DatabaseName object. You should first create the object, then pass the name as string. For example:

```suggestion
            $db = DatabaseName::tryFromValue($GLOBALS['db']);
            $this->relation->fixPmaTables($db !== null ? $db->getName() : '');
```